docs/includes/fundamentals/read-operations/ReadOperationsTest.php

```diff
@@ -94,4 +101,35 @@ public function testFirst(): void
         $this->assertNotNull($movie);
         $this->assertInstanceOf(Movie::class, $movie);
     }
+
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testText(): void
+    {
+        // start-text
+        $movies = Movie::where('$text', ['$search' => '"love story"'])
+            ->get();
+        // end-text
+
+        $this->assertNotNull($movies);
+        $this->assertCount(2, $movies);
+    }
+
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testTextRelevance(): void
+    {
+        // start-text-relevance
+        $movies = Movie::where('$text', ['$search' => '"love story"'])
```

Comment:
Same as the other test:

```suggestion
        $movies = Movie::where('$text', ['$search' => 'love story'])
```